docs/web_reference.rst

```diff
@@ -1473,7 +1473,7 @@ duplicated like one using :meth:`Application.copy`.
 
       :param Application subapp: nested application.
 
-      :returns: a :class:`MatchedSubAppResource` instance.
+      :returns: a :class:`PrefixedSubAppResource` instance.
```

Comment:
This is wrong, it returns a `MatchedSubAppResource`: https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_app.py#L268

Reply:
ok, had some qustions about this. `MatchedSubAppResource` goes to ignore